network-api/networkapi/mozfest/templates/mozfest/mozfest-base.html

```diff
@@ -2,7 +2,9 @@
 {% load wagtailcore_tags wagtailimages_tags homepage_tags card_tags wagtailmetadata_tags custom_image_tags i18n static %}
 
 
-{% block web_monetization %}{% endblock %}
+{% block web_monetization %}
+  <meta name="monetization" content="$ilp.uphold.com/aKpM9kwnGBy4">
```

Comment:
Is there somewhere that this value can be found for future engineering work? Also, would it make sense to turn this into an environment variable like `COIL_WALLET_URL` or is this essentially a "permanent" public URL?

Reply:
Hi @Pomax!

From Anil: It looks like the URL can be found on the coil site when logged in, and does not appear to change.

<img width="1676" alt="image (3)" src="https://user-images.githubusercontent.com/18314510/151040070-8f7bebfb-0b54-40be-b432-e639736c8c2d.png">